core/src/main/resources/hudson/model/Computer/index.jelly

```diff
@@ -28,17 +28,14 @@ THE SOFTWARE.
     <st:include page="sidepanel.jelly" />
     <!-- no need for additional breadcrumb here as we're on an index page already including breadcrumb -->
     <l:main-panel>
-      <st:include page="index-top.jelly" it="${it}" />
-
-      <!-- temporarily offline switch -->
-      <div style="float:right">
+      <l:app-bar title="">
```

Comment:
Shouldn't there be a title?

Reply:
yes the agent heading should be moved into the title (Built-in Node etc)

Reply:
And where shall be placed the 'status' icon? Or is there some way to place the icon in the app-bar-title too?
I try it before, and I can add the icon only as item (like the buttons) and in that case it is aligned to right side, with looks very weird.

Reply:
Status icons are not recommended to be in the heading so fine to remove as part of this:

https://weekly.ci.jenkins.io/design-library/Symbols/

> Don't add symbols to headings, they just create visual clutter

Reply:
somthing like this mnight be a good compromis
![image](https://user-images.githubusercontent.com/89339813/211797795-0165a26d-db41-4c8b-9673-38b3b1f2585e.png)

Reply:
thanks can you update the screenshots in the PR description? I'll try test it out later on my only worry which needs checking is that other agent types will have defined the title in the `index-top.jelly`

Reply:
Screen shots are up-to-date.